server/accounts.go

```diff
@@ -4114,18 +4120,65 @@ type transform struct {
 	src, dest string
 	dtoks     []string
 	stoks     []string
-	dtpi      []int8
+	dtpi      [][]int // destination token position indexes
+	dtpinp    []int32 // destination token position index number of partitions
 }
 
-// Helper to pull raw place holder index. Returns -1 if not a place holder.
-func placeHolderIndex(token string) int {
-	if len(token) > 1 && token[0] == '$' {
-		var tp int
-		if n, err := fmt.Sscanf(token, "$%d", &tp); err == nil && n == 1 {
-			return tp
+func getMappingFunctionArgs(functionRegEx *regexp.Regexp, token string) []string {
+	commandStrings := functionRegEx.FindStringSubmatch(token)
+	if len(commandStrings) > 1 {
+		return commaSeparatorRegEx.Split(commandStrings[1], -1)
```

Comment:
could maybe be made into a single regex call using `FindAllStringSubmatch()` but need to do some experimentation to be sure :)

Reply:
Maybe, but IMHO probably not much to gain from that, at least right now I re-use that commaSeparatorRegEx for all the commands. At this point I would leave it how it is and later when I have more time I could do some experimentation and eventually create another PR if there's an improvement to be had.